build.gradle

```diff
@@ -20,6 +20,12 @@ if (ext.isFips) {
 } else {
     ext.awsLcGitVersionId = 'v1.30.1'
 }
+
+// Check for user inputted git version ID.
+if (System.properties["AWSLC_REF"]) {
```

Comment:
`REF` is a somewhat ambiguous qualifier and doesn't align with the nomenclature used by the variable you're assigning to. I'd prefer something like `AWSLC_GITVERSION` which is totally unambiguous.

Reply:
Updated!